spring/spring4/spring4-remote/src/test/java/org/infinispan/spring/provider/SpringRemoteCacheManagerFactoryBeanContextTest.java

```diff
@@ -1,6 +1,6 @@
 package org.infinispan.spring.provider;
 
-import static org.junit.Assert.assertEquals;
+import static org.testng.Assert.assertEquals;
```

Comment:
We should use `AssertJUnit`, which is part of TestNG but keeps the same parameters as the JUnit `Assert`